x-pack/test/api_integration/deployment_agnostic/apis/observability/ai_assistant/complete/product_docs/products.ts

```diff
@@ -0,0 +1,525 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { ProductDocumentationAttributes } from '@kbn/product-doc-common/src/documents';
+
+interface Inference {
+  inference_id: string;
+  model_settings: {
+    task_type: string;
+  };
+  chunks: Array<{
+    text: string;
+    embeddings: Record<string, number>;
+  }>;
+}
+
+interface SemanticFieldWithInference {
+  text: string;
+  inference?: Inference;
+}
+
+interface SemanticArrayFieldWithInference {
+  text: string[];
+  inference?: Inference;
+}
+
+type ProductDocumentation = Omit<
+  ProductDocumentationAttributes,
+  'content_body' | 'ai_summary' | 'ai_questions_answered'
+> & {
+  content_body?: SemanticFieldWithInference;
+  ai_summary?: SemanticFieldWithInference;
+  ai_questions_answered?: SemanticArrayFieldWithInference;
+};
+
+export const mockKibanaProductDoc: ProductDocumentation = {
+  content_title: 'Lens',
+  content_body: {
+    text: '## Lens\nTo create a visualization, drag the data fields you want to visualize to the workspace, then Lens uses visualization best practices to apply the fields and create a visualization that best displays the data.\n\nWith Lens, you can:\n\n- Create area, line, and bar charts with layers to display multiple indices and chart types.\n- Change the aggregation function to change the data in the visualization.\n- Create custom tables.\n- Perform math on aggregations using Formula.\n- Use time shifts to compare the data in two time intervals, such as month over month.\n- Add annotations and reference lines.',
+    inference: {
+      inference_id: '.elser-2-elasticsearch',
+      model_settings: { task_type: 'sparse_embedding' },
+      chunks: [
```

Comment:
The only fields I'd expect in a product doc artifacts is`content_title` and `content_body`. Nothing related to inference. This should be handled in ES.